src/object_store/src/object/opendal_engine/opendal_object_store.rs

```diff
@@ -374,6 +378,20 @@ impl OpendalStreamingUploader {
             if self.abort_on_err {
                 self.writer.abort().await?;
             }
+            match self.op.stat(&self.path).await {
+                Ok(metadata) => {
+                    tracing::info!(
+                        "The file {:?} can be stated, but fail when writer write. Metadata: {:?}",
+                        self.path,
+                        metadata,
+                    );
+                }
+                Err(e) => tracing::info!(
+                    "The file {:?} can not be stated, and fail when writer write. Err {:?}",
+                    self.path,
+                    e
+                ),
```

Comment:
```suggestion
                Err(e) => tracing::info!(
                    path = ?self.path,
                    error = %e.as_report(),
                    "File can not be stated, and fail when writer write.",
                ),
```

Can refer to https://www.notion.so/risingwave-labs/A-Guide-to-Error-Handling-that-Just-Works-Part-I-e108f8d0845b4bddaa0e5843791a8cac?pvs=4#f781cf9ecaee436a97adf9f051f2ce99